libs/core/src/components/pds-divider/pds-divider.scss

```diff
@@ -1,11 +1,11 @@
 :host {
-  --background-color: var(--pine-color-neutral-grey-400);
-  --offset-xxs: var(--pine-spacing-xxs);
-  --offset-xs: var(--pine-spacing-xs);
-  --offset-sm: var(--pine-spacing-sm);
-  --offset-md: var(--pine-spacing-md);
-  --offset-lg: var(--pine-spacing-lg);
-  --offset-xxl: var(--pine-spacing-xxl);
+  --color-background: var(--pine-color-grey-400);
+  --offset-xxs: var(--pine-spacing-050);
+  --offset-xs: var(--pine-spacing-100);
+  --offset-sm: var(--pine-spacing-200);
+  --offset-md: var(--pine-spacing-300);
+  --offset-lg: var(--pine-spacing-450);
+  --offset-xxl: var(--pine-spacing-800);
```

Comment:
How do you feel about prepending `spacing` as the category here?

Reply:
agreed. This one was done before our conversation. LOL